### ISSUE TITLE
No audio (ICE failure?)

### ISSUE THREAD

**Issue by brianjmurrell** [BelledonneCommunications/liblinphone#185]
In this [log](http://brian.interlinx.bc.ca/linphone-no-audio-ice-failure.txt) there are two incoming calls, neither of which had any audio.

When I clicked the call stats icon on the top right it said that ICE connectivity was _In progress_ and never moved beyond that.

I am using linphone-android **80fa401d5 - (tag: 4.6.0-alpha)** with **SDK 5.0.1-pre.6+0253a33**.

**Comment by brianjmurrell** [comment]
Any ideas here?  This failure seems to make linphone unusable when it's behind any IPv4 NAT.  :frowning:

**Comment by wananbayue** [comment]
@brianjmurrell I have the same problem, but my SIP server is opensips. Have you solved this problem?

**Comment by brianjmurrell** [comment]
@wananbayue No.  I have not.  As you can see, this issue is still open.  As you can also see it has been +1d by 3 others so we are not the only ones seeing it.

This issue also has no comments from any project maintainer.
